bdb-ip.md

```diff
@@ -0,0 +1,437 @@
+# Improvement proposals for HTTP API
+
+
+Currently, there is two milestones grouping issues for upcoming HTTP API
+endpoints:
+
+- [Changes that break the existing HTTP API of BDB
+  v0.11](https://github.com/bigchaindb/bigchaindb/milestone/11)
+- [Changes that are additions to the exiting HTTP API of BDB
+  v0.11](https://github.com/bigchaindb/bigchaindb/milestone/9)
+
+After releasing BDB v1.0, breaking changes to the external APIs are not
+permitted anymore. Additions to the APIs are however always allowed.
```

Comment:
Should we add an exclusion here - permitted for bugfixes and security considerations?

One more point to note might be that:
Suppose we change something in BigchainDB API, and make the corresponding changes to the (officially) supported drivers, can we claim that as long as the driver API (with the user programs) remain the same, the change is permissible. That might give us more leeway going forward if we come up with something unexpected.